conn/physic/example_test.go

```diff
@@ -393,6 +393,30 @@ func ExampleRelativeHumidity() {
 	// 20%rH
 }
 
+func ExampleRelativeHumidity_Set() {
+	var r physic.RelativeHumidity
+
+	if err := r.Set("50.6%rH"); err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println(r)
+
+	if err := r.Set("20%rH"); err != nil {
```

Comment:
You can put "20%" in the example, to show that it works?

Reply:
sure can.